test/consumer.integration.spec.ts

```diff
@@ -28,6 +28,66 @@ describe('FFI integration test for the HTTP Consumer API', function () {
     value,
   });
 
+  describe('with matching rules', function () {
+    beforeEach(function () {
+      pact = makeConsumerPact(
+        'matcher-consumer',
+        'matcher-provider',
+        FfiSpecificationVersion['SPECIFICATION_VERSION_V3']
+      );
+
+      const interaction = pact.newInteraction('matcher test');
+      interaction.uponReceiving('a write request');
```

Comment:
Just to improve clarity
```suggestion
      interaction.uponReceiving('a request to /test');
```